ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -503,6 +503,12 @@ def _get_columns(
                             arr_data_type = re.match(
                                 r"(?:\w*)(?:[(]*)(\w*)(?:.*)", str(column["type"])
                             ).groups()
+                            if isinstance(arr_data_type, list) or isinstance(
+                                arr_data_type, tuple
+                            ):
+                                arr_data_type = ColumnTypeParser.get_column_type(
```

Comment:
@ayush-shah @ulixius9 can we double check this, as it will apply to all the connectors.

Reply:
@harshach tested this with hive, big query, snowflake, mysql & mssql its working fine